services/web/server/src/simcore_service_webserver/functions/_functions_service.py

```diff
@@ -42,7 +40,6 @@
 router = RPCRouter()
 
 
-@router.expose(reraise_if_error_type=(UnsupportedFunctionClassError,))
```

Comment:
also in this file all the functions were removed form being exposed to RPC. why sis this the case?

Reply:
correct